packages/hub/src/scope.ts

```diff
@@ -115,7 +115,7 @@ export class Scope implements ScopeInterface {
    * @inheritDoc
    */
   public setUser(user: User | null): this {
-    this._user = normalize(user);
+    this._user = user || {};
```

Comment:
The implementation here was incorrect from the beginning. `normalize` returned `any`, so we did `this._user = <any>`, which shouldn't be allowed, as `this._user` is a non-optional `User` type. Also, `clear` sets it to `{}` not `null`.